src/components/Tray/TrayButton.tsx

```diff
@@ -23,22 +17,7 @@ export const TrayButton: FC<{
         marginLeft: 0,
       }}
     >
-      <Canvas
-        style={{
-          width: icon.width,
-          height: icon.heigth,
-        }}
-      >
-        {iconSVG && (
-          <ImageSVG
-            svg={iconSVG}
-            x={0}
-            y={0}
-            width={icon.width}
-            height={icon.heigth}
-          />
-        )}
-      </Canvas>
+      <Icon source={trayIcons[props.label]} />
     </Pressable>
   )
```

Comment:
Mi sono accorto che anche le icone nella Tray sono allineate un po' male, probabilmente lo erano anche prima.
Magari si fixa in un altra branch se è effettivamente un bug. Probabilmente basta aggiungere qualche opzione di alignment

Reply:
lo erano decisamente anche prima #159, potrei fixarle adesso in effetti